api/app/Http/Controllers/AuthController.php

```diff
@@ -212,7 +217,52 @@ public function handleProviderCallback($provider, Socialite $socialite, UserRepo
     }
 
     /**
-     * Check so the provider exists.
+     * Provide a requester with user information for single sign on.
+     *
+     * @param  string  $requester
+     * @param  Request $request
+     *
+     * @return Response
+     */
+    public function singleSignOn($requester, Request $request)
+    {
+        if ($token = $request->cookie('JwtAuthToken')) {
+            $user = $this->jwtAuth->toUser($token);
+        } else {
+            $user = null;
```

Comment:
Ah, that makes sense. Can you add a comment explaining that, as this looks like an exceptional case, but it makes sense that Vanilla can't cope with that.